### PR TITLE
Some REST routes

### PR DESCRIPTION
**Pull request by ouziel-slama**

### CODE REVIEW

**Comment by ouziel-slama** [comment]
Add REST endpoint optimized for tables impacted by the migration to log-structured database:
    * GET /addresses/<address>/balances
    * GET /assets/<asset>/
    * GET /assets/<asset>/balances
    * GET /assets/<asset>/orders
    * GET /orders/<tx_hash>
    * GET /orders/<tx_hash>/matches

**Comment by ouziel-slama** [comment]
I will update the `Documentation` repo with these new routes.

File: counterparty-lib/counterpartylib/lib/api.py [hunk 1]
```diff
@@ -24,6 +24,7 @@
 import struct
 import apsw
 import flask
+from flask import request
```
[the remark this answers is not in the document]
Comment (adamkrellenstein) [reply counterparty-lib/counterpartylib/lib/api.py:27]: this sounds silly to me. quiet this warning?

File: counterparty-lib/counterpartylib/lib/api.py [hunk 2]
```diff
@@ -1010,6 +1011,35 @@
...
+        def handle_address_balances(address):
+            return ledger.get_address_balances(self.db, address)
+
+        @app.route('/assets/<asset>/balances', methods=['GET'])
+        def handle_asset_balances(asset):
+            return ledger.get_asset_balances(self.db, asset)
+
+        @app.route('/assets/<asset>/', methods=['GET'])
+        def handle_asset_info(asset):
+            return get_asset_info(asset=asset)
```
[the remark this answers is not in the document]
Comment (adamkrellenstein) [reply counterparty-lib/counterpartylib/lib/api.py:1026]: is this real?

**Review by adamkrellenstein — APPROVED** [review]

File: counterparty-lib/counterpartylib/lib/api.py (hunk 2 above)
[the remark this answers is not in the document]
Comment (ouziel-slama) [reply counterparty-lib/counterpartylib/lib/api.py:1026]: nope! `asset` is used in a binding!
there is Pylint issue, will take care of all them together.. and then we could focus on each new one..

File: counterparty-lib/counterpartylib/lib/api.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (ouziel-slama) [reply counterparty-lib/counterpartylib/lib/api.py:27]: same! that should be cleaned I think :-) just to respect the standard.. will take some days for Pylint asap!

**Pull request merged**